hdf5.spec

```
@@ -248,8 +257,13 @@ ln -s %{_javadir}/junit.jar java/lib/junit.jar
 ln -s %{_javadir}/slf4j/api.jar java/lib/slf4j-api-1.7.25.jar
 ln -s %{_javadir}/slf4j/nop.jar java/lib/ext/slf4j-nop-1.7.25.jar
 ln -s %{_javadir}/slf4j/simple.jar java/lib/ext/slf4j-simple-1.7.25.jar
+
 # Fix test output
+%if (0%{?suse_version} >= 1500)
+junit_ver=$(sed -n '/<version>/{s/^.*>\([0-9]\.[0-9]*\)<.*/\1/;p;q}' /usr/share/maven-poms/junit.pom)
+%else
 junit_ver=$(sed -n '/<version>/{s/^.*>\([0-9]\.[0-9]*\)<.*/\1/;p;q}' /usr/share/maven-poms/JPP-junit.pom)
+%endif
```

For future reference it's usually best to make the difference needed in branches such as this one as small as possible to mitigate future copy-and-paste update errors.  I.e.

```suggestion
%if (0%{?suse_version} >= 1500)
junit_ver_file=junit
%else
junit_ver_file=JPP-junit
%endif
junit_ver=$(sed -n '/<version>/{s/^.*>\([0-9]\.[0-9]*\)<.*/\1/;p;q}' /usr/share/maven-poms/$junit_ver_file.pom)

```